sitback-furniture/src/components/counter/counter.jsx

```diff
@@ -0,0 +1,14 @@
+import styles from './counter.module.scss';
+
+const Counter = ({count,onCountUpdate})=>{
+
+    return(
+        <div className={styles['counter-button-wrapper']}>
```

Comment:
Do we really need div here?

Reply:
yes, the outer wrapper is needed to wrap the button and count elements and also to add stylings for the component.